jobs/pgbackup/templates/bin/pgrestore

```diff
@@ -76,7 +76,23 @@ S3_ENDPOINT="<%= p('pgbackup.s3.endpoint') %>"
 
 # extra 'cat' in the pipeline is necessary because pg_restore closes stdin
 # which gof3r does not expect, and consequently panics...
+
+#if the name of the single database is set, we assume that we will restore the same database that we backup before
+single_database_name=""
+single_database_name="<%= p('pgbackup.pgdump.database') %>"
+if [[ $single_database_name != "" ]]; then
```

Comment:
if pgdump.database is set so we assume that we want to perform a single database backup